src/lib.rs

```diff
@@ -891,36 +889,23 @@ impl Drop for BufferWriteMapping {
     }
 }
 
-pub struct BufferAsyncMapping<T> {
-    pub data: T,
-    buffer_id: wgc::id::BufferId,
-}
-//TODO: proper error type
-pub type BufferMapAsyncResult<T> = Result<BufferAsyncMapping<T>, ()>;
-
-impl<T> Drop for BufferAsyncMapping<T> {
```

Comment:
Where do we do the buffer unmapping now if it doesn't happen during `Drop`?

Reply:
I believe that `BufferAsyncMapping` was left over from when `map_read` and `map_write` used callbacks, and is now completely unused. So this `Drop` impl is also unused.

Reply:
Ok sounds good, thanks!